### PR TITLE
Update system.d

### PR DESCRIPTION
**Pull request by adamasantares**
add short example

### CODE REVIEW

File: std/system.d [hunk 1]
```diff
@@ -14,7 +14,20 @@ immutable
 {
     /++
         Operating system.
-
+        
+        Example:
+        ```
```
Comment (wilzbach) [inline std/system.d]: Unfortunately the Markdown syntax isn't supported by DDoc - have a look at this page:

https://dlang.org/spec/ddoc.html

or other files in Phobos. I looks roughly like this:

````
Example:
----
struct A
{
    int a;
    mixin(bitfields!(
        uint, "x",    2,
        int,  "y",    3,
        uint, "z",    2,
        bool, "flag", 1));
}
A obj;
obj.x = 2;
obj.z = obj.x;
----
```

Comment (adamasantares) [inline std/system.d]: ah, ok, thank you. So I just have to remove md syntax to pass the chacks?

**Comment by lodo1995** [comment]
@adamasantares You have to remove trailing spaces and hard tabs for the autotester to be happy.

File: std/system.d [hunk 2]
```diff
@@ -14,7 +14,20 @@ immutable
 {
     /++
         Operating system.
-
+        
+        Example:
+        ----
+        import std.system, std.stdio;
+        int main(string[] args) {
```
Comment (quickfur) [inline std/system.d]: Please use [Phobos style](http://dlang.org/dstyle.html) for all code examples.

Specifically, all opening braces `{` should be on their own line, and 4-space indentation should be used.

Comment (adamasantares) [inline std/system.d]: ok

Comment (adamasantares) [inline std/system.d]: hm, is it real to replace tabs by 4 spaces via github editor?

**Comment by adamasantares** [comment]
ah, I did it

**Comment by lodo1995** [comment]
LGTM after squash.

@adamasantares you should squash your commits into one, so that the history is kept clean (having seven commit to add an example is not very good).

**Comment by adamasantares** [comment]
@lodo1995 I agree, but can't find how to realize it by github UI. I haven't local copy of repository.

**Comment by lodo1995** [comment]
@adamasantares I strongly suggest you to work on a local copy and use command-line git to perform the squash and update this PR.
I guess the only doable alternative is to create a new branch, apply the final modifications to it, close this PR and open a new one with the new branch. But of course, if you are then forced to apply other changes, this method does not scale well.

**Comment by adamasantares** [comment]
@lodo1995 ok, I'll close this pull request and then create new.

**Pull request closed without merging**